tests/server/test_build_logger.py

```diff
@@ -167,14 +168,110 @@ def test_build_logger(yaml_config, raises_error, default_logger, num_function_lo
     assert isinstance(logger, AsyncLogger)
     assert isinstance(logger.logger, MultiLogger)
     if default_logger:
-        assert isinstance(logger.logger.loggers[0], PythonLogger)
+        assert isinstance(logger.logger.loggers[0].logger.loggers[0], PythonLogger)
         return
     assert len(logger.logger.loggers) == num_function_loggers + 1
 
-    # check for system logger
+    # check for default system logger behaviour
     system_logger = logger.logger.loggers[-1]
     assert system_logger.target_identifier == (
-        f"category:{MetricCategories.SYSTEM.value}"
+        f"category:{MetricCategories.SYSTEM.value}/prediction_latency"
```

Comment:
do the pipeline logs all have `category:` prepended to them? where does this happen again?

Reply:
See the comment above, introduced in https://github.com/neuralmagic/deepsparse/pull/697/commits/780b38aa685a33d934a82a36dc641d8f2ab91db1